packages/editor-ui/src/views/TemplatesCollectionView.vue

```diff
@@ -1,220 +0,0 @@
-		<template #header>
-			<div v-if="!notFoundError" :class="$style.wrapper">
-				<div :class="$style.title">
-					<n8n-heading v-if="collection && collection.name" tag="h1" size="2xlarge">
-						{{ collection.name }}
-					</n8n-heading>
-					<n8n-text v-if="collection && collection.name" color="text-base" size="small">
-						{{ $locale.baseText('templates.collection') }}
-					</n8n-text>
-					<n8n-loading :loading="!collection || !collection.name" :rows="2" variant="h1" />
-				</div>
-			</div>
-			<div v-else :class="$style.notFound">
-				<n8n-text color="text-base">{{
-					$locale.baseText('templates.collectionsNotFound')
-				}}</n8n-text>
-			</div>
-		</template>
-		<template v-if="!notFoundError" #content>
-			<div :class="$style.wrapper">
-				<div :class="$style.mainContent">
-					<div v-if="loading || (collection && collection.description)" :class="$style.markdown">
-						<n8n-markdown
-							:content="collection && collection.description"
-							:images="collection && collection.image"
-							:loading="loading"
-						/>
-					</div>
-					<TemplateList
-						:infinite-scroll-enabled="false"
-						:loading="loading"
-						:use-workflow-button="true"
-						:workflows="loading ? [] : collectionWorkflows"
-						@useWorkflow="onUseWorkflow"
-						@openTemplate="onOpenTemplate"
-					/>
-				</div>
-				<div :class="$style.details">
-					<TemplateDetails
-						:block-title="$locale.baseText('template.details.appsInTheCollection')"
-						:loading="loading"
-						:template="collection"
-					/>
-				</div>
-			</div>
-		</template>
-	</TemplatesView>
-</template>
-
-<script lang="ts">
-import { defineComponent } from 'vue';
-import { mapStores } from 'pinia';
-
-import TemplateDetails from '@/components/TemplateDetails.vue';
-import TemplateList from '@/components/TemplateList.vue';
-import TemplatesView from './TemplatesView.vue';
-
-import { workflowHelpers } from '@/mixins/workflowHelpers';
-import type {
-	ITemplatesCollection,
-	ITemplatesCollectionFull,
-	ITemplatesWorkflow,
-	ITemplatesWorkflowFull,
-} from '@/Interface';
-
-import { setPageTitle } from '@/utils/htmlUtils';
-import { VIEWS } from '@/constants';
-import { useTemplatesStore } from '@/stores/templates.store';
-import { usePostHog } from '@/stores/posthog.store';
-import { useTemplateWorkflow } from '@/utils/templates/templateActions';
-import { useExternalHooks } from '@/composables/useExternalHooks';
-import { useNodeTypesStore } from '@/stores/nodeTypes.store';
-
-export default defineComponent({
-	name: 'TemplatesCollectionView',
-	components: {
-		TemplateDetails,
-		TemplateList,
-		TemplatesView,
-	},
-	mixins: [workflowHelpers],
-	setup() {
-		const externalHooks = useExternalHooks();
-
-		return {
-			externalHooks,
-		};
-	},
-	computed: {
-		...mapStores(useTemplatesStore, usePostHog),
-		collection(): ITemplatesCollectionFull | null {
-			return this.templatesStore.getCollectionById(this.collectionId);
```

Comment:
seems like we need to clean up the stores and api as well, no?

Reply:
Agree, there was some code in there that is used to setup templates but I think it can be moved to new store. Will remove those also